### ISSUE TITLE
About Factory Function and jQuery / MooTools support

### ISSUE THREAD

**Issue by SamBrishes** [pytesNET/tail.select#137]
Hellow,

I'm playing with the idea to remove the jQuery and MooTools implementation completely because they don't really bring us big benefits. They both also just return an array with all available select instances, so if you want to change or interact with one you need either re-catch the instance again or pick it out from the array. There is also no "jQuery" way to interact / handle with all select fields at once anyway. 

The only difference is the initialization... And it isn't a big deal using `tail.select( $("select") )` instead of `$(select).tailselect()` (which also looks really awful due to the missing point). The same goes for MooTools (which is also a really unused library in general).

If jQuery is really important, maybe an own plugin would be way better. This would also allow building up a more "jQuery" known behaviour (such as interact with all created tail.select instances at once).

What do you think?

Sincerely,
Sam.

**Comment by Calak** [comment]
I agree, you reduce the code and the the workaround isn't a big deal.

PS: can you accept/reject my PR, then I can keep working.
